backend/src/main.ts

```diff
@@ -1,13 +1,18 @@
+import { AppModule } from './app.module';
 import { NestFactory } from '@nestjs/core';
 import { WsAdapter } from '@nestjs/platform-ws';
 
-import { AppModule } from './app.module';
+const NODE_ENV = process.env.NODE_ENV ?? "development";
+const PORT = process.env.PORT ?? 3000;
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.useWebSocketAdapter(new WsAdapter(app));
 
-  await app.listen(process.env.PORT ?? 3000);
+  await app.listen(PORT);
+  if(NODE_ENV==="development") {
+    console.log(`\nNow listening @ port ${PORT}`);
```

Comment:
You should use `Logger.log` here instead of `console.log` - the `Logger` class from nest is a global singleton that uses the configured logger & keeps the output consistently formatted regardless of how logging is configured.

Also - it is probably okay to keep this regardless of `NODE_ENV`. Logs are useful when there are issues in production, and logging what address/port we're listening on can be helpful to troubleshoot issues with accepting traffic, etc.